src/vs/code/electron-sandbox/issue/issueReporterService.ts

```diff
@@ -1110,6 +1184,33 @@ export class IssueReporter extends Disposable {
 		}
 	}
 
+	private setLoading() {
+		// Show loading
+		this.previewButton.label = 'Loading Extension Data...';
+		this.previewButton.enabled = false;
+
+		const extensionDataCaption = this.getElementById('extension-id')!;
+		hide(extensionDataCaption);
+
+		const showLoading = this.getElementById('ext-loading')!;
+		show(showLoading);
+
+		const iconElement = document.createElement('span');
+		iconElement.classList.add(...ThemeIcon.asClassNameArray(Codicon.loading), 'codicon-modifier-spin');
+		showLoading.append(iconElement);
+	}
```

Comment:
for all of this, I think we should just have:

```
[x] Include Additional Extension info 🔄
```
and then
```
[x] Include Additional Extension info (show)
```

Reply:
![Recording 2023-10-23 at 12 05 14](https://github.com/microsoft/vscode/assets/54879025/d32575a7-2094-4635-9af1-918e81d51a9b)

It's a tad bit wonky, but since this code here specifies a specific scheme for the `showInfo` type, I didn't want to repeat code just for this case. It prevents us from doing this:

```
<span ....>
<a href>...
<span...>,
```

https://github.com/microsoft/vscode/blob/49b6fa9a20c1fbbfb4db343c63b68d0941d3d69b/src/vs/code/electron-sandbox/issue/issueReporterService.ts#L259